src/sentry/api/paginator.py

```diff
@@ -45,6 +45,16 @@ class BadPaginationError(Exception):
     pass
 
 
+class MissingPaginationError(Exception):
+    def __init__(self, func_name):
+        self.func_name = func_name
+        super().__init__(
+            f"""Response is not paginated correctly in {func_name}.
+                         List API response should be paginated otherwise it can break the product in future caused by eventual growth.
+                        Learn more about pagination in https://develop.sentry.dev/api/concepts/#paginating-responses and reach out to #discuss-api if you have any questions."""
+        )
```

Comment:
you can actually make this a string template:
```suggestion
class MissingPaginationError(Exception):
    error_message: str = """Response is not paginated correctly in {func_name}.
                     List API response should be paginated, as lack of pagination can break the product in the future due to eventual growth.
                     Learn more about pagination in https://develop.sentry.dev/api/concepts/#paginating-responses and reach out to #discuss-api if you have any questions."""

    def __init__(self, func_name: str):
        self.func_name = func_name
        super().__init__(self.error_message.format(func_name=func_name))
```